x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/datastream_hooks.tsx

```diff
@@ -43,7 +43,7 @@ export function useIndexTemplateExists(
         path: `/api/index_management/index_templates/${templateName}`,
         method: 'get',
       }),
-    { enabled: enabled || !!templateName }
+    { enabled }
```

Comment:
the || !!templateName here meant the request was always being sent even when disabled as templateName is always provided. Originally I had it optional.